File: lesson_12/Homework_12_6/product/HotDrink.java
package lesson_12.Homework_12_6.product;

public class HotDrink extends Drink{
private int temperature;

public HotDrink(String name, int volume, int temperature) {
        super(name, volume);
        this.temperature = temperature;
    }


    public int getTemperature() {
        return temperature;
    }

    public void setTemperature(int temperature) {
        this.temperature = temperature;
    }   
   
    @Override
    public String toString() {
        return String.format("%s, volume %d, temperature %d", name, volume, temperature);

    }

}

File: lesson_13/hometask/HW_py/HotDrink.py
from Drink import Drink

class HotDrink(Drink):

    def __init__(self, name, volume, temperature):
        self._name = name
        self._volume = volume
        self._temperature = temperature

    def getTemp(self):
        return self._temperature
    
    def setTemperature(self, temp):
        self._temperature = temp
    
    def __str__(self):
        return f"{self._name} volume {self._volume} temperature {self._temperature}" 

